.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "public/themes"]
 	path = public/themes
 	url = git://github.com/openSUSE/opensuse-themes.git
+
+[submodule "public/chameleon"]
+	path = public/chameleon
+	url = git@github.com:openSUSE-zh/chameleon.git
```

Comment:
It is for all openSUSE sites. Now it has been ported to software.opensuse.org and wiki. More prots will be made in future.